Makefile

```
@@ -490,7 +490,7 @@ go-postgres-unit-tests: build-prep test-prep
 shell-unit-tests:
 	@echo "+ $@"
 	$(SILENT)mkdir -p shell-test-output
-	bats --print-output-on-failure --verbose-run --recursive --report-formatter junit --output shell-test-output scripts
+	bats --print-output-on-failure --verbose-run --recursive --report-formatter junit --output shell-test-output scripts tests/e2e
```

It might be good to see the directories searched more clearly:
```suggestion
	bats --print-output-on-failure --verbose-run --recursive --report-formatter junit --output shell-test-output \
		scripts \
		tests/e2e
```